sky/task.py

```diff
@@ -968,6 +968,23 @@ def sync_storage_mounts(self) -> None:
                     self.update_file_mounts({
                         mnt_path: blob_path,
                     })
+                elif store_type is storage_lib.StoreType.AZURE:
+                    if isinstance(storage.source,
+                                  str) and data_utils.is_az_container_endpoint(storage.source):
```

Comment:
```suggestion
                    if (isinstance(storage.source,
                                  str) and data_utils.is_az_container_endpoint(storage.source)):
```
and re-run formatter

Reply:
done